.gitignore

```diff
@@ -3,6 +3,7 @@ target-tiny/
 lcov.info
 *.profraw
 .soroban/
+.DS_Store
```

Comment:
We can add this, but recommend you adding this to your personal global gitignore file since it's platform specific. A global gitignore file is a great place to put anything about your computer.

Reply:
Ah ok, fair point